jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -28,85 +29,93 @@
         element.refresh();
         return element.asData();
     }
-
+    
     /**
      * Check that all elements meet condition
+     *
      * @param condition to compare
      * @return DataListAssert
      */
     @JDIAction("Assert that each of '{name}' elements meet condition")
-    public DataListAssert<T, D> each(JFunc1<D, Boolean> condition) {
-        jdiAssert(LinqUtils.all(data(), condition::execute), Matchers.is(true));
+    public DataListAssert<T, D> each(JFunc1<D, Boolean> condition, String... messages) {
+        jdiAssert(LinqUtils.all(data(), condition::execute), Matchers.is(true), messages);
         return this;
     }
-
+    
     /**
      * Check that at least one element meets condition
+     *
      * @param condition to compare
      * @return DataListAssert
      */
     @JDIAction("Assert that any of '{name}' elements meet condition")
-    public DataListAssert<T, D> any(JFunc1<D, Boolean> condition) {
-        jdiAssert(LinqUtils.any(data(), condition::execute), Matchers.is(true));
+    public DataListAssert<T, D> any(JFunc1<D, Boolean> condition, String... messages) {
+        jdiAssert(LinqUtils.any(data(), condition::execute), Matchers.is(true), messages);
         return this;
     }
-
+    
     /**
      * Check that only one of elements meets condition
+     *
      * @param condition to compare
      * @return DataListAssert
      */
     @JDIAction("Assert that only one of '{name}' elements meet condition")
-    public DataListAssert<T, D> onlyOne(JFunc1<D, Boolean> condition) {
-        jdiAssert(single(data(), condition::execute), Matchers.is(notNullValue()));
+    public DataListAssert<T, D> onlyOne(JFunc1<D, Boolean> condition, String... messages) {
+        jdiAssert(single(data(), condition::execute), Matchers.is(notNullValue()), messages);
         return this;
     }
-
+    
     /**
      * Check that none of elements meets condition
+     *
      * @param condition to compare
      * @return DataListAssert
      */
     @JDIAction("Assert that none of '{name}' meet condition")
-    public DataListAssert<T, D> noOne(JFunc1<D, Boolean> condition) {
-        jdiAssert(first(data(), condition::execute), Matchers.is(nullValue()));
+    public DataListAssert<T, D> noOne(JFunc1<D, Boolean> condition, String... messages) {
+        jdiAssert(first(data(), condition::execute), Matchers.is(nullValue()), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)